js/iframeResizer.js

```diff
@@ -201,7 +201,7 @@
 				}
 
 				function checkSingle(){
-					var remoteHost  = settings[iframeId].remoteHost;
+					var remoteHost  = settings[iframeId] && settings[iframeId].remoteHost;
```

Comment:
Doesn't this make `remoteHost` a boolean? Wouldn't that break the lines below: `return origin === remoteHost`